src/listeners/message.ts

```diff
@@ -83,32 +83,101 @@ client.on("message", async message => {
+
+            collector.on("collect", async reaction => {
+                const cancelEmbed = new Discord.MessageEmbed()
+                    .setColor(errorColor)
+                    .setAuthor("outgoingCancelled", "global")
+                if (reaction.emoji.name === "❎") {
+                    msg.reactions.cache.get("❎")!.users.remove()
+                    msg.reactions.cache.get("✅")!.users.remove()
+
+                    msg.edit(cancelEmbed)
+                }
+                else {
+                    db.collection("users").updateOne({ id: message.author.id }, { $set: { staffMsgTimestamp: message.createdTimestamp } })
+                    const staffMsg = new Discord.MessageEmbed()
+                        .setColor(neutralColor)
+                        .setAuthor("Incoming message from " + message.author.tag)
+                        .setDescription(message.content)
+                        .addField("To reply", `\`+dm ${message.author.id} \``)
+
+                    const dmEmbed = new Discord.MessageEmbed()
+                        .setColor(successColor)
+                        .setAuthor(getString("outgoing", "global"))
+                        .setDescription(message.content)
+                        .setFooter(getString("outgoingDisclaimer", "global"))
+                    if (message.attachments.size > 1) {
+                        const images: (Discord.BufferResolvable | Stream)[] = []
+                        message.attachments.forEach(file => images.push(file.attachment))
+                        staffMsg.setTitle("View attachments")
+                        dmEmbed.setTitle(getString("attachmentsSent", "global"))
+                        staffBots.send({ embed: staffMsg, files: images })
+                        return message.channel.send(dmEmbed)
+                    } else if (message.attachments.size > 0) {
+                        staffMsg
+                            .setTitle("View attachment")
+                            .setImage(message.attachments.first()!.url)
+                        dmEmbed.setTitle(getString("attachmentSent", "global"))
+                        staffBots.send(staffMsg)
+                    } else staffBots.send(staffMsg) //staff-bots
+                    msg.reactions.cache.get("❎")!.users.remove()
+                    msg.reactions.cache.get("✅")!.users.remove()
+
+                    msg.edit(dmEmbed)
+                }
+            })
+
+            collector.on("end", () => {
+                const timeOutEmbed = new Discord.MessageEmbed()
+                    .setColor(errorColor)
+                    .setAuthor(getString("timedOut", "global"))
+                    .setDescription(getString("resendInfo", "global"))
+                
+                    msg.edit(timeOutEmbed)
```

Comment:
Following the pattern on previous timeout actions, just edit the content of the message without sending a new embed. Also change the string a bit